firmware/application/apps/pocsag_app.cpp

```diff
@@ -168,21 +170,23 @@ void POCSAGAppView::on_packet(const POCSAGPacketMessage* message) {
         /*
         // Too many errors for reliable decode.
         if (pocsag_state.errors >= 3) {
-            console.write("\n\x1B\x0D" + prefix + " Too many decode errors.");
+            console.write("\n" + STR_COLOR_MAGENTA + prefix + " Too many decode errors.");
```

Comment:
It's up to you, but you can still rely on the preprocessor to combine the literals here too. Not sure if clang-format will like it though.